libloki/proof-of-work.js

```diff
@@ -0,0 +1,70 @@
+const hash = require('js-sha512');
+const bb = require('bytebuffer');
+
+// Increment Uint8Array by 1 with carrying
+function incrementNonce(nonce) {
+  idx = nonce.length - 1;
```

Comment:
```suggestion
  let idx = nonce.length - 1;
```
this is javascript danger-zone :)
> [no var] will look up the scope chain until it finds the variable or hits the global scope